tools/sense_studio/static/sense_studio_logo.svg

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
In SenseStudio I would like to use a logo without background, because for example if you hover the button on the top left, the surrounding background color changes and reveals that white box:

![image](https://user-images.githubusercontent.com/5707898/107953074-e1ddfa80-6f9a-11eb-9173-20e21a1e8f65.png)

And we don't support dark-mode on SenseStudio yet, so there should be no problem.